### DISCUSSION TITLE
Neo4j Support

### DISCUSSION THREAD

**Discussion by kaiserbergin** [testcontainers/testcontainers-dotnet#579] [Ideas]
Are there any plans to support neo4j's graph database? If not, would it be something you'd be willing to accept as a contribution to the repo?

**Comment by HofmeisterAn** [comment]
Far as I know, no one is working on a Neo4j module. A contribution would be great. Please keep these #493 limitations in mind. We can migrate the Neo4j module afterwards. I am working right now on the new module structure.

**Reply by kaiserbergin** [reply]
Thanks, I'll get on it. Absolutely love this project as it has really simplified our Integration Tests locally and in build pipelines.
